include/seqan3/alphabet/quality/all.hpp

```diff
@@ -61,17 +61,42 @@
  * | Solexa              | Solexa, Illumina [1.0; 1.8[ | Phred+64 | seqan3::phred68legacy | [-5 .. 62]        | [0 .. 67]  | [';' .. '~'] |
  *
  * The most distributed format is the *Sanger* or <I>Illumina 1.8+</I> format.
- * Despite typical Phred scores for Illumina machines range from 0 to maximal
- * 41, it is possible that processed reads reach higher scores. If you don't
- * intend to handle Phred scores larger than 41, we recommend to use
- * seqan3::phred42 due to its more space efficient implementation.
- * For other formats, like Solexa and Illumina 1.0 to 1.7 the type
- * seqan3::phred68legacy is provided. To cover also the Solexa format, the Phred
- * score is stored as a <B>signed</B> integer starting at -5.
- * If you want to store PacBio HiFi reads, we recommend to use seqan3::phred94, as these use the full range of the phred
- * quality scores.
- * An overview of all the score formats and their encodings can be found here:
- * https://en.wikipedia.org/wiki/FASTQ_format#Encoding (last access 01.12.2020).
+ * Despite typical Phred scores for Illumina machines range from 0 to maximal 41, it is possible that processed reads
+ * reach higher scores. If you don't intend to handle Phred scores larger than 41, we recommend using seqan3::phred42
+ * due to its more space-efficient implementation (see below). If you want to store PacBio HiFi reads, we recommend to
+ * use seqan3::phred94, as these use the full range of the Phred quality scores.
+ * For other formats, like Solexa and Illumina 1.0 to 1.7 the type seqan3::phred68legacy is provided. To cover also the
+ * Solexa format, the Phred score is stored as a <B>signed</B> integer starting at -5.
```

Comment:
I think we have markdown,

```suggestion
 * Solexa format, the Phred score is stored as a **signed** integer starting at -5.
```

can you double check?